### ISSUE TITLE
A11y Update: Bubble Component

### ISSUE THREAD

**Issue by gabew1984** [openedx/paragon#1500]
Bubble component "warning" variant fails WCAG text contrast requirement of 4.5:1 

https://www.w3.org/TR/WCAG22/#contrast-minimum

Update "warning" variant to use "gray-900" for text and icon inside Bubble.

**Comment by adamstankiewicz** [comment]
Completed via https://github.com/edx/brand-edx.org/pull/44
